src/Dofs/sparsity_pattern.jl

```diff
@@ -233,3 +287,41 @@ function _condense_sparsity_pattern!(K::SparseMatrixCSC{T}, dofcoefficients::Vec
 
     return nothing
 end
+
+for (func,                              pre_f,                                                                                      inner_f,                                                                                                                                                                    return_values) in (
+    (:cross_element_coupling_count,     :(max_buffer_length = 0;),                                                                  :(max_buffer_length += coupling_length),                                                                                                                                    :(max_buffer_length)),
+    (:cross_element_coupling,           :(I = Vector{Int}(undef, max_buffer_length); J = Vector{Int}(undef, max_buffer_length)),    :(I[cnt+1:cnt+coupling_length] = repeat(cell_dofs,length(neighbour_unique_dofs)); J[cnt+1:cnt+coupling_length] = repeat(neighbour_unique_dofs,inner=length(cell_dofs))),    :(I, J)),
+)
+    @eval begin
+        function $(func)(dh::AbstractDofHandler, topology::ExclusiveTopology, full_cross_element::Bool; max_buffer_length::Int = 0)
+            $(pre_f)
+            element_dof_start = 0
+            cnt = 0
+            for (fhi, fh) in pairs(dh.fieldhandlers)
+                
+                for fi in fh.field_interpolations
+                    if(!(typeof(fi)<:DiscontinuousLagrange) && !full_cross_element)
```

Comment:
We might want to think about this type of branching further. I have thought about adding a traits system to the interpolations to mark them e.g. as "L2-conforming", so users can add new interpolations (with corresponding dispatches), add the trait and things will just work out of the box. What do you think? cc @fredrikekre

Reply:
Yes, something more extensible that `typeof(fi)<:DiscontinuousLagrange` would be to prefer (like traits as already mentioned). Something similar to https://docs.julialang.org/en/v1/manual/interfaces/:

```
IsDisontinuous(::Type) = false
...
IsDisontinuous(::Type{<:DiscontinuousLagrange}) = true

if IsDisontinuous(typeof(fi)) ...
```